paper/paper.md

```diff
@@ -56,22 +64,22 @@ Here, we propose a software `buildH` to fill this need. `buildH` is very light a
 
-All structure and trajectory reading / writing are dealt with the MDAnalysis module [Michaud-Agrawal2011; Gowers2016]. Mathematical vector operations are dealt with Numpy [@Harris2020] and accelerated with Numba [@Lam2015]. This allows to get very decent performances although the software is written in pure Python. For example, the reconstruction of all Hs and order parameter calculation on a trajectory of 2500 frames with 128 POPC molecules can be handled in approximately 7 minutes using a single core Xeon processor at 3.60 GHz, whereas it was almost 30 minutes long without Numba.
+All structure and trajectory reading / writing are dealt with the MDAnalysis module [@Michaud-Agrawal2011; @Gowers2016]. Mathematical vector operations are dealt with Numpy [@Harris2020] and accelerated with Numba [@Lam2015]. This allows to get very decent performances although the software is written in pure Python. For example, the reconstruction of all Hs and order parameter calculation on a trajectory of 2500 frames with 128 POPC molecules can be handled in approximately 7 minutes using a single core Xeon processor at 3.60 GHz, whereas it was almost 30 minutes long without Numba.
 
 `buildH` has been implemented with good practices of software development in mind [@Jimenez2017; @Taschuk2017]:
 
-- version control repository on GitHub (https://github.com/patrickfuchs/buildH),
+- version control repository on GitHub [https://github.com/patrickfuchs/buildH](https://github.com/patrickfuchs/buildH),
 - open-source license (BSD-3-Clause),
 - continuous integration through tests,
-- and documentation (https://buildh.readthedocs.io/).
+- and documentation [https://buildh.readthedocs.io/](https://buildh.readthedocs.io/).
 
-Some notebooks are present on the the github repository which explain how `buildH` works and how to analyze data produced by it. In case of trouble, any user can post an issue.
+Some notebooks are present on the the github repository which explain how `buildH` works and how to analyze  the data produced. In case of trouble, any user can post an issueon the GitHub section.
```

Comment:
One extra space to remove between `analyze` and `the`.
Add a space between `issue` and `on`